zerver/lib/push_notifications.py

```diff
@@ -251,6 +254,42 @@ def dedupe_device_tokens(
     return result
 
 
+class APNsResultInfoDict(TypedDict):
+    successfully_sent: bool
+    delete_device_id: NotRequired[int]
+    delete_device_token: NotRequired[str]
```

Comment:
I'd usually use a dataclass for this, not a TypedDict.

Reply:
Done